cookbooks/aws-parallelcluster-slurm/templates/default/slurm/login_nodes_daemon_service/login_nodes_daemon_config.json.erb

```diff
@@ -0,0 +1,4 @@
+{
+  "termination_script_path": "<%= node['cluster']['shared_dir'] %>/login_nodes_on_termination_script.sh",
+  "termination_message": "System will be terminated after <%= GRACETIME_PERIOD %> minutes."
```

Comment:
I think that specifying the gracetime period in the termination message is misleading because the user would not now when exactly the system will shut down.
We have two options here:
1. Leave the message generic: "The system will be terminated in less than ${GRACETIME_PERIOD} minutes"
2. Be more specific: "The system will be terminated on ${TERMINATION_DATE_TIME}", where *TERMINATION_DATE_TIME* is computed at runtime when the termination is scheduled.

Since we are close to the deadline, if you find hard to inject the termination date and time, I suggest to go with the generic approach and refine the solution in a post-beta follow up PR.

Reply:
That makes sense. I use the generic approach for now.